### PR TITLE
[Tabs][add information][1491921]

### PR DESCRIPTION
**Pull request by Rajeshwari-v**
Added information. Please review.

### CODE REVIEW

File: msteams-platform/tabs/design/tabs-mobile.md [hunk 1]
```diff
@@ -130,3 +130,7 @@ For authentication to work on mobile clients, you must upgrade you Teams JavaScr
 ### Low bandwidth and intermittent connections
 
 Mobile clients regularly need to function with low bandwidth and intermittent connections. Your app should handle any timeouts appropriately by providing a contextual message to the user. You should also user progress indicators to provide feedback to your users for any long-running processes.
+
+> [!NOTE]
+> Tabs are enabled on mobile only after the application is added to an allow list, based on the input of the approval team. 
+> To check mobile responsiveness, reach out to teamsubm@microsoft.com.
```
Comment (KirtiPereira) [inline msteams-platform/tabs/design/tabs-mobile.md:136]: Are there multiple "allow list" or is there only one such list? Please confirm and modify usage of "article" accordingly.
```suggestion
> [!NOTE]
> Tabs are enabled on mobile only after the application is added to an allow list. The application is added to an allow list based on the input of the approval team. 
> To check mobile responsiveness, reach out to teamsubm@microsoft.com. 
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@Rajeshwari-v - please verify and incorporate the comments and suggestions. Add the SME to review the PR.

File: msteams-platform/tabs/design/tabs-mobile.md (hunk 1 above)
Comment (Rajeshwari-v) [reply msteams-platform/tabs/design/tabs-mobile.md:136]: Allow list is single. Confirmed with SME.

**Comment by KirtiPereira** [comment]
@ramendhe - kindly review and provide your comments / approval for the PR. Thank you.

**Review by ramendhe — APPROVED** [review]

**Review by KirtiPereira — APPROVED** [review]
Approve

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR.

**Pull request merged**